05-classification.Rmd

```diff
@@ -400,33 +415,18 @@ cancer %>%
   slice(1:5) # subset the first 5 rows
 ```
 
-From this, we see that 3 of the 5 nearest neighbours to our new observation are
-malignant so classify our new observation as malignant. We circle those 5 in
-the plot below:
+Below we show in mathematical detail how the `mutate` step was used to compute the `dist_from_new` 
+variable (the distance to the new observation) for each of the 5 nearest neighbours in the training
+data.
 
 
-```{r 05-multiknn-3, echo = FALSE, fig.cap="Scatterplot of concavity versus perimeter with 5 nearest neighbours circled"}
-perim_concav + annotate("path",
-  x = new_point[1] + 1.4 * cos(seq(0, 2 * pi,
-    length.out = 100
-  )),
-  y = new_point[2] + 1.4 * sin(seq(0, 2 * pi,
-    length.out = 100
-  ))
-)
-```
-</center>
-
 ```{r 05-multiknn-4, echo = FALSE}
 my_distances <- table_with_distances(cancer[, attrs], new_point)
 neighbours <- my_distances[order(my_distances$Distance), ]
 k <- 5
 tab <- data.frame(neighbours[1:k, ], cancer[order(my_distances$Distance), ][1:k, c("ID", "Class")])
 ```
 
-It can be difficult sometimes to read code as math, so here we mathematically
-show the calculation of distance for each of the 5 closest points.
-
 |Perimeter | Concavity | Distance          | Class          |
```

Comment:
This is a table that we should have a number and title for, and reference in the text.